test/e2e/e2e_leaderelection_test.go

```diff
@@ -35,9 +35,12 @@ import (
 	utilpointer "k8s.io/utils/pointer"
 	"sigs.k8s.io/descheduler/cmd/descheduler/app/options"
 	"sigs.k8s.io/descheduler/pkg/descheduler"
+	"sigs.k8s.io/descheduler/pkg/framework/plugins/pluginbuilder"
 )
 
 func TestLeaderElection(t *testing.T) {
+	pluginbuilder.PluginRegistry = pluginbuilder.NewRegistry()
+	pluginbuilder.RegisterDefault(pluginbuilder.PluginRegistry)
```

Comment:
When writing e2e tests, sometimes, depending on the test, we will have to remember to register plugins